tests/sentry/grouping/fingerprint_inputs/fingerprint-on-sdk.json

```diff
@@ -0,0 +1,14 @@
+{
+  "_fingerprinting_rules": [
+    {
+      "matchers": [["sdk", "sentry.javascript.nextjs"]],
+      "fingerprint": ["sdk-nextjs"]
+    }
+  ],
+  "logentry": {
+    "formatted": "Es Dee Kay"
+  },
+  "sdk": {
+    "name": "sentry.javascript.nextjs"
```

Comment:
Cool. I see it working
<img width="439" alt="image" src="https://github.com/getsentry/sentry/assets/44410/fc1bb3f2-fac0-4a52-a5ea-4c356c30fa23">